web/ITechArt.StudentLabs/ITechArt.StudentsLab.PresentationLayer/Controllers/FeedbackController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using ITechArt.StudentsLab.PresentationLayer.Models;
+using System.Collections.Generic;
+using ITechArt.StudentsLab.BusinessLayer.Contracts;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using BlFeedbackDateModel = ITechArt.StudentsLab.BusinessLayer.Models.FeedbackDateModel;
```

Comment:
For Presentation Layer if you use Request or Response or ViewModel postfixes you do not have such issues